### PR TITLE
fix(binanceus) - ratelimits 3

### PR DESCRIPTION
**Pull request by ttodua**
completed all endpoints and RLs , according to : https://docs.binance.us/

the next step would be exceptions (error) mappings corrections for binance-us specifically, but i'll get to that after exception-mapping-PR is merged.

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei the only thing i don't understand is the travis error happening there. i can't reproduce it locally.

**Comment by ttodua** [comment]
ahh, reproduced now. 
let me find fix for it.

**Comment by ttodua** [comment]
strange though, what i see is that, it inherits all implicit api declarations from binance (which is expected) in php abstract file,  and many overriden endpoints (i.e. `exchangeInfo` or etc) are only declared once (which is also expected), however, only specific endpoints (i.e.`sub-account/spotSummary`) are repeated two times

**Comment by ttodua** [comment]
@carlosmiei PR ready.

File: build/generateImplicitAPI.js [hunk 1]
```diff
@@ -104,8 +104,11 @@ function createImplicitMethods(){
     const exchanges = Object.keys(storedCamelCaseMethods);
     for (const index in exchanges) {
         const exchange = exchanges[index];
-        const camelCaseMethods = storedCamelCaseMethods[exchange];
-        const underscoreMethods = storedUnderscoreMethods[exchange]
+        let camelCaseMethods = storedCamelCaseMethods[exchange];
+        let underscoreMethods = storedUnderscoreMethods[exchange];
+        // remove duplicates
+        camelCaseMethods = [... new Set(camelCaseMethods)];
```
Comment (carlosmiei) [inline build/generateImplicitAPI.js:110]: @ttodua I'm not sure this is the right place to fix the issue, maybe we should not have duplicated endpoints in exchanges that are parent<->child?

Comment (ttodua) [reply build/generateImplicitAPI.js:110]: yeah right, seems i've just applied "painkiller" instead of solution. let me fix it better

Comment (ttodua) [reply build/generateImplicitAPI.js]: issue was in other place, this script didnt need modification.
Now PR is ready, whenever you can review.

File: ts/src/binanceus.ts [hunk 2]
```diff
@@ -100,6 +102,106 @@ export default class binanceus extends binance {
...
+                        'order/cancelReplace': 1,
+                        'order/oco': 1,
+                    },
+                    'delete': {
+                        'order': 1,
+                        'openOrders': 1,
+                        'orderList': 1,
+                    },
+                },
+                'sapi': {
```
Comment (carlosmiei) [inline ts/src/binanceus.ts:127]: I'm getting some "not found" errors

```
n binanceus sapiGetApiTradingStatus
Debugger attached.
2024-01-26T09:45:18.178Z
Node.js: v20.7.0
CCXT v4.2.21
binanceus.sapiGetApiTradingStatus ()
ExchangeNotAvailable binanceus GET https://api.binance.us/sapi/v1/apiTradingStatus?timestamp=1706262319131&recvWindow=10000&signature=803944d95d6cbad84cccb5102deabaf549bfb195f28f6f67ea7a1505e51a1d6a 404  {"timestamp":1706262319414,"status":404,"error":"Not Found","message":"No message available","path":"/sapi/v1/apiTradingStatus"}
```
```
n binanceus privateGetStatus                       
Debugger attached.
2024-01-26T09:45:51.849Z
Node.js: v20.7.0
CCXT v4.2.21
binanceus.privateGetStatus ()
ExchangeNotAvailable binanceus GET https://api.binance.us/api/v3/status?timestamp=1706262352827&recvWindow=10000&signature=2326ef3588894c352107b7d2d6e93303a379ec85d37e6598bad7c1ac455c9b97 404 Not Found <html><body><h2>404 Not found</h2></body></html>
```

Comment (ttodua) [reply ts/src/binanceus.ts:127]: eventually, i had to revise all endpoints, as I had several misplaced ones. now all corrected.

Comment (carlosmiei) [reply ts/src/binanceus.ts:127]: going to check it

**Pull request merged**